Enum style (#1731)
GMLC-TDC/HELICS: docs/user-guide/simultaneous_cosimulations.md

The manual approach works fine. All the network core types accept user specified port numbers. The following script will start up two brokers on separate port numbers:

```sh

> HelicsBroker --CoreType=zmq --port=20200 &

Author: ```suggestion
helics_broker --core_type=zmq --port=20200 &
```

> HelicsBroker --CoreType=zmq --port=20400 &

Author: ```suggestion
helics_broker --core_type=zmq --port=20400 &
```

helics_broker_server --zmq --duration=24hours
```

> It will also generate brokers as needed so the `HelicsBroker` does not need to be restarted for every run.

Author: ```suggestion
It will also generate brokers as needed so the `helics_broker` does not need to be restarted for every run.
```

If there are multiple users and you want to verify that a specific broker can only be used with federates you control. It is possible to add a key to the broker that is required to be supplied with the federates to connect to the broker. **NOTE:** _this is not a cryptographic key, it is just a string that is not programmatically accessible to others._

```sh

> HelicsBroker --CoreType=zmq --key=my_broker_key

Author: ```suggestion
helics_broker --core_type=zmq --key=my_broker_key
```
